src/kernels/jupyter/launcher/jupyterExecution.ts

```diff
@@ -190,6 +193,32 @@ export class JupyterExecutionBase implements IJupyterExecution {
         return Promise.resolve(undefined);
     }
 
+    public async validateRemoteUri(uri: string): Promise<void> {
```

Comment:
Added a function to validate the URI. Might use this in the kernel finder too for cached items.